packages/buckets/src/api/index.ts

```diff
@@ -53,9 +53,9 @@ import {
 import { Context, ContextInterface } from '@textile/context'
 import { GrpcConnection } from '@textile/grpc-connection'
 import { WebsocketTransport } from '@textile/grpc-transport'
-import CID from 'cids'
 import { EventIterator } from 'event-iterator'
 import log from 'loglevel'
+import { CID } from 'multiformats'
```

Comment:
Switching to multiformity over cids due to some release regressions. This should be low impact, but if it is an issue, we can revert. It _will_ require a minor version bump. The change_should_ be backward compatible, but it is an external dev change, so we might want to consider just doing a major bump to be on the safe side.